test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/DotnetCliUtil.cs

```diff
@@ -76,26 +76,44 @@ public static string GetXplatDll()
 #else
                 "Release";
 #endif
+                var configurationDirectory = Path.Combine(dir.FullName, "artifacts", "NuGet.CommandLine.XPlat", "bin", configuration);
+                var referenceTfm = new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.NetCoreApp, new Version(int.MaxValue, 0, 0));
+                var bestTfm = GetTfmToCopy(configurationDirectory, referenceTfm);
+                var filePath = Path.Combine(configurationDirectory, bestTfm, XPlatDll);
 
-                var relativePaths = new string[]
+                if (File.Exists(filePath))
```

Comment:
Should we log the path for the file we dynamically found?

Reply:
Given that it's run in test code, we'd need to plumb the test logger in every test this is called in order for the logs to be shown in the output.
Logging on standard out doesn't do anything.